app/Http/Controllers/Api/AbstractApiController.php

```
@@ -175,6 +175,16 @@ protected function noContent()
     }
 
     /**
+     * Respond with a 401 unauthorized.
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    protected function unauthorized()
+    {
+        return $this->setStatusCode(401)->respond();
```

This does not look correct. You should throw an exception.